charts/longhorn-1.1/longhorn/100.0.1+up1.1.3-rc2/Chart.yaml

```diff
@@ -0,0 +1,40 @@
+annotations:
+  catalog.cattle.io/auto-install: longhorn-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0'
```

Comment:
can we also add the following 2 annotations:
```
catalog.cattle.io/kube-version:  <the same value as the kubeVersion in chart.yaml> 
catalog.cattle.io/upstream-version: <longhorn's version>
```

Reply:
Thanks for the info about this new change. Added.